pkgs/by-name/mo/mobsql/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, buildGoModule, fetchFromSourcehut }:
+
+buildGoModule rec {
+  pname = "mobsql";
+  version = "0.6.0";
+
+  src = fetchFromSourcehut {
+    owner = "~mil";
+    repo = "mobsql";
+    rev = "v${version}";
+    hash = "sha256-jPx/We8a1L9hLDgxxg/pAfBkbT+cl7l8EmruYZ4maug=";
+  };
+
+  proxyVendor = true;
```

Comment:
Why do you need it?

Reply:
Weird, I recall it fail to fetch the vendored modules without that but can no longer reproduce.  I removed that line now.